server/src/main/java/com/hedera/block/server/Server.java

```diff
@@ -59,7 +60,9 @@ public static void main(final String[] args) throws IOException {
         // Init BlockNode Configuration
         final Configuration configuration =
                 ConfigurationBuilder.create()
-                        .withSource(SystemEnvironmentConfigSource.getInstance())
+                        .withSource(
+                                new BlockNodeMappedConfigSource(
+                                        SystemEnvironmentConfigSource.getInstance()))
```

Comment:
Noted. 

When I was researching your proposed solution and was trying to understand how the `MappedConfigSource` works, I stumbled on some usages and indeed found what you are saying, to use the class directly [(see here)](https://github.com/hashgraph/hedera-services/blob/develop/platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/config/internal/ConfigMappings.java), but thought that instead of doing it with a utility class and since the `MappedConfigSource` is not final, I thought that inheritance would package the initialization of the configs in a good way. In any case I think we need to do some maintenance since we need to provide the mappings for what we want to support as env variables that need a different key at runtime. In theory this of course could be done dynamically, but I see no point in doing so, increased complexity and redundant operations to map keys that we would not use anyway. That being said, I think that it is a good idea to somehow do a test that would fail if we introduce a new config class and do not extend the Junit test that would check that all the mappings are correct. 

So let me know what you think, I could switch to a utility class and use the `MappedConfigSource` directly, or I could leave like that. Thanks!

Reply:
After a conversation with @jsync-swirlds, he noted that for the functionality we are going to need to use the `MappedConfigSource`, it is planned to be supported by default. So in reality we are just making a workaround. That being said, I agree completely that making the new class is completely redundant, since we are expecting that it will be removed in the future. Will fall back to using `MappedConfigSource` directly and make some tests.